spock-specs/src/test/groovy/org/spockframework/smoke/parameterization/DataTables.groovy

```diff
@@ -389,6 +389,83 @@ a | b | c
     4 || 5 | 9
   }
 
+  def "two or more underscores can be used to separate multiple data tables"() {
```

Comment:
I'm not seeing a test, there the tables have different row counts.

Reply:
I could add one, but as the table columns are translated to data pipes, I thought this is already covered by the existing tests that check for data providers with different sizes.
It is the same as if you for example have one data table with 4 rows and then a data pipe with only 3 values.

Reply:
I think it is better to have some redundant tests instead of having missed something. And just because it is transformed in a certain way today, does not mean it will be so in the future.

Reply:
I didn't mean no test is necessary because of the columns being transformed to data pipes, but because columns are data providers "just like data pipes" and all data providers have to have the same length.
But sure, I'll add another test, no problem.